Topshelf.Linux/packages.config

```diff
@@ -15,5 +14,6 @@
 // specific language governing permissions and limitations under the License.
 -->
 <packages>
-	<package id="Topshelf" version="3.2.0" targetFramework="net40" />
+  <package id="Mono.Posix" version="4.0.0.0" targetFramework="net40" />
```

Comment:
We should avoid dependency on Mono.Posix.. as it is not binary compatible between builds available on MS.NET, and the binary included with mono linux' packages. This makes pretty hard to build/deploy the same binaries capable of working on both platforms (MS.NET & Mono/Linux).

May you add to MonoHelper.cs all P/Invoke code required for Unix Signals management?

Regards